Sources/AppStoreConnectCLI/Commands/TestFlight/BetaGroups/RemoveTestersFromGroupCommand.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+
+struct RemoveTestersFromGroupCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "removeuser",
+        abstract: "Remove beta testers from a beta group")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(help: "Name of TestFlight beta tester group")
+    var groupName: String
+
+    @Argument(help: "Beta testers' email addresses")
+    var emails: [String]
+
+    func validate() throws {
+        if emails.isEmpty {
+            throw ValidationError("Missing expected argument '<emails>'.")
```

Comment:
Maybe something like: "Expected at least one email" is more accurate?
You can provide more detailed help on `emails` so that it calls the individual values `email` if you wish